linuxdeploy-plugin-gtk.sh

```diff
@@ -180,7 +181,7 @@ cat > "$HOOKFILE" <<\EOF
 
 gsettings get org.gnome.desktop.interface gtk-theme 2> /dev/null | grep -qi "dark" && GTK_THEME_VARIANT="dark" || GTK_THEME_VARIANT="light"
 APPIMAGE_GTK_THEME="${APPIMAGE_GTK_THEME:-"Adwaita:$GTK_THEME_VARIANT"}" # Allow user to override theme (discouraged)
-CACHEDIR="$(mktemp --tmpdir --directory .AppRun.XXXXXXXX)"
```

Comment:
See https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/issues/22#issuecomment-997190886

You must be including `/tmp` in the template if you are interested in POSIX compliancy. The standard template which is implicitly used is a full path, too. See https://pubs.opengroup.org/onlinepubs/009695399/functions/mktemp.html.

Reply:
Good catch!